ver/us/symbol_addrs.txt

```diff
@@ -7732,7 +7732,7 @@ D_802483EC_7F30FC = 0x802483EC; // type:data rom:0x7F30FC
 D_80248F18_7F3C28 = 0x80248F18; // type:data rom:0x7F3C28
 D_80248F20_7F3C30 = 0x80248F20; // type:data rom:0x7F3C30
 D_80248F28_7F3C38 = 0x80248F28; // type:data rom:0x7F3C38
-D_8024A632_7F5342 = 0x8024A632; // type:data rom:0x7F5342
+mac_00_dojoBattleIDs = 0x8024A632; // type:data rom:0x7F5342
```

Comment:
We moved this symbol 2 bytes earlier, so we need to adjust the addresses for this:
```suggestion
mac_00_dojoBattleIDs = 0x8024A630; // type:data rom:0x7F5340
```